lib/core/src/animation/physics/inertia.rs

```diff
@@ -119,23 +124,37 @@ impl KinematicsProperties {
 // === Getters ===
 
 impl KinematicsProperties {
-    pub fn velocity    (&self) -> Vector3<f32> { self.velocity }
-    pub fn acceleration(&self) -> Vector3<f32> { self.acceleration }
-    pub fn mass        (&self) -> f32          { self.mass }
+    /// `Velocity` getter.
+    pub fn velocity(&self) -> Vector3<f32> {
+        self.velocity
+    }
+
+    /// `Acceleration` getter.
+    pub fn acceleration(&self) -> Vector3<f32> {
+        self.acceleration
+    }
+
+    /// `Mass` getter.
```

Comment:
I'm sad we cannot disable the requirement of writing docs for getters :P It's sad.